packages/sample-app/src/plugin-sdk-addons/shared-modules.ts

```diff
@@ -0,0 +1,29 @@
+import type {
```

Comment:
For some reason github will not let me comment on the above SVG (or the PNG bin), but one thing we may want to consider is adding rules/readme for contributing SVGs. Several properties and elements that are outputted by the sketch tool are not actually required in the SVG source (including comments etc).

Reply:
@florkbr 

> For some reason github will not let me comment on the above SVG (or the PNG bin)

Yup, because GitHub only supports line comments, not file comments :smiley: there should be a button like "comment on this file" or similar.

> Several properties and elements that are outputted by the sketch tool are not actually required in the SVG source (including comments etc)

I'd say that in general, files in `packages/sample-app/src/images` should be production-ready (i.e. optimized) by default. If there are SVG processing tools we could use in our webpack build, we can use them.

As for these two specific images:
- `packages/sample-app/src/images/favicon.png` - copied from https://github.com/patternfly/patternfly-react-seed/blob/master/src/favicon.png
- `packages/sample-app/src/images/pfColorLogo.svg` - copied from `packages/sample-app/node_modules/@patternfly/react-core/src/demos/examples/pfColorLogo.svg`